playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -56,6 +56,19 @@
     src: inventory-caasp4.yml.j2
     dest: "{{ socok8s_workspace }}/inventory/caasp4.yml"
 
+- name: Extend inventory to co-use the LB node as OSH deployer node
+  copy:
+    content: "{{ soc_aio_overrides }}"
```

Comment:
Im also not sure what this really does?? Its doing something completely out of scope for the caasp role plus its only for a specific deployment but it will affect both deployments...why not have this somewhere else if needed?

Reply:
for this, kvm-deploy_caasp.yml seems the perfect play to move this to. Its out of the role, its specific to kvm, and its clear to what is doing.

Reply:
Yeah, I first tried to implement it inside a new kvm-deploy_ccp_deployer.yml task. But that added so much boilerplate that I didn't really like it. I guess I need to revisit that decision :)

Reply:
> for this, kvm-deploy_caasp.yml seems the perfect play to move this to. Its out of the role, its specific to kvm, and its clear to what is doing.

I've now move it to it's now playbook (kvm-deploy_ccp_deployer.yml). Mainly to closely match openstack-* step. But I can also add it to the kvm-deploy_caasp if you prefer it that way.